client/utils/format-ssr.js

```diff
@@ -125,7 +125,7 @@ Taxonomies: Product Types: ${ printTerms( systemStatus.settings.taxonomies ) }
 Taxonomies: Product Visibility: ${ printTerms(
 		systemStatus.settings.product_visibility_terms
 	) }
-Connected to WooCommerce.com: ${
+Connected to Woo.com: ${
 		systemStatus.settings.woocommerce_com_connected === 'yes'
```

Comment:
Umm, I guess this is similar to the "WCpay" mentions in code after the branding rollout, that we don't want to touch them? 🤔

Reply:
So, I think this is a string that will be added to the `SSR` - however, I think you are probably right because maybe people have macros etc with the search for this string built into them, so I've reverted this change in db2cf9c67.